fk/teamjoin.go

```diff
@@ -60,6 +61,12 @@ func teamJoin(teamUUID uuid.UUID) exitCode {
 	printCheckboxSuccess(action)
 	out.Print("\n\n")
 
+	db.RecordRequestToJoinTeam(team.RequestToJoinTeam{
+		UUID:        teamUUID,
```

Comment:
nope, `UUID` is the **request** UUID. This has tripped us up a few times so I propose we make it explicit by renaming `UUID` to `RequestUUID`